src/gg.regression.unity.bots/Runtime/Scripts/RGOverlay/RGSequenceManager.cs

```diff
@@ -91,6 +94,11 @@ public void Awake()
 
     public void Start()
     {
+        if (RGUtils.IsMobile())
```

Comment:
In relation to the comment I make at the end of this file, I suppose what I am thinking might be `if (RGUtils.IsMobile() && is in portrait mode, i.e. screen height is larger than screen width)`